code/modules/mob/living/simple_animal/abnormality/_tools/he/theonite_slab.dm

```diff
@@ -0,0 +1,76 @@
+#define STATUS_EFFECT_SLAB /datum/status_effect/slab
+/obj/structure/toolabnormality/theonite_slab
+	name = "theonite slab"
+	desc = "A slab, made out of a seamless mixture of stone and metal. It's covered in runes, and bloody spikes erupt from the centerpiece."
+	icon_state = "slab"
+	var/list/users = list()
+
+/obj/structure/toolabnormality/theonite_slab/attack_hand(mob/living/carbon/human/user)
+	..()
+	if(!do_after(user, 6))
```

Comment:
```suggestion
	if(!do_after(user, 6, src))
```
Stops the user spamming it for rapid stacks, despite being funny.

Reply:
feature. Just like it's a feature for skin prophecy